CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We renamed the "Body Text" CSL bibliography header format name to "Text body" as per internal LibreOffice conventions. [#13074](https://github.com/JabRef/jabref/pull/13074)
 - We moved the "Modify bibliography title" option from the CSL styles tab of the Select Style dialog to the OpenOffice/LibreOffice side panel and renamed it to "Bibliography properties". [#13074](https://github.com/JabRef/jabref/pull/13074)
 - We changed path output display to show the relative path with respect to library path in context of library properties. [#13031](https://github.com/JabRef/jabref/issues/13031)
+- We made two constructors for PdfMergeDialog to call with and without a BibEntry. For this to work we also added 2 helper methods to initialize the dialog and finalize it. [#13031](https://github.com/JabRef/jabref/issues/13132)
```

Comment:
CHANGELOG.md entries sholud be **user facing**

The number 13031 is toally wrong, re-use the number of the link https://github.com/JabRef/jabref/issues/13132.

```suggestion
- When importing a PDF, there is no empty entry column shown in the multi merge dialog. [#13132](https://github.com/JabRef/jabref/issues/13132)
```